### PR TITLE
git: Add more shortcuts

### PR DESCRIPTION
**Pull request by linsui**
I remove the note because openssh.json is out of date. git-with-ssh should be a better choice. (Related: https://github.com/lukesampson/scoop/issues/2414)

### CODE REVIEW

File: bucket/git-with-openssh.json [hunk 1]
```diff
@@ -33,30 +32,37 @@
...
+            "cmd\\gitk.exe",
+            "gitk"
         ]
     ],
     "post_install": "git config --global credential.helper manager",
     "checkver": {
-        "url": "https://github.com/git-for-windows/git/releases/latest",
-        "re": "v(?<version>[\\d\\w.]+)/PortableGit-(?<short>[\\d.]+).*\\.exe"
+        "github": "https://github.com/git-for-windows/git",
+        "regex": "v([\\d\\w.]+)/PortableGit-(?<ver>[\\d\\w.]+)-64-bit"
```
Comment (Ash258) [inline bucket/git-with-openssh.json]: `\w` already contains `\d`

```suggestion
        "regex": "v([\\w.]+)/PortableGit-(?<ver>[\\w.]+)-64-bit"
```

image

File: bucket/git-with-openssh.json [hunk 2]
```diff
@@ -33,30 +32,37 @@
         [
             "git-bash.exe",
             "Git Bash",
-            "--cd-to-home",
-            "usr\\share\\git\\git-for-windows.ico"
+            "--cd-to-home"
+        ],
+        [
+            "cmd\\git-gui.exe",
+            "Git Gui"
```
Comment (Ash258) [inline bucket/git-with-openssh.json]:
```suggestion
            "Git GUI"
```

image

**Comment by h404bi** [comment]
@linsui 
What's the use case of the gitk shortcut?

**Comment by linsui** [comment]
@h404bi Sorry, I didn't notice that gitk should be called in cli. This shortcut should be removed.

**Pull request merged**